test/tools/unified-spec-runner/match.ts

```diff
@@ -294,7 +318,7 @@ export function specialCheck(
     // $$sessionLsid
     const session = entities.getEntity('session', expected.$$sessionLsid, false);
     expect(session, `Session ${expected.$$sessionLsid} does not exist in entities`).to.exist;
-    const entitySessionHex = session.id!.id.buffer.toString('hex').toUpperCase();
+    const entitySessionHex = session.id!.id.buffer.toString().toUpperCase();
```

Comment:
Will open a new ticket to document this change. In summary, the Buffer.toString() does not take an argument, so this change doesn't modify any functionality.